### DISCUSSION TITLE
How to set Name for tree element

### DISCUSSION THREAD

**Discussion by gdams** [wixtoolset/issues#6435] [Questions] [answered]
I've been running the accessibility insights tool against my Wix installer and it's reporting the following error:

`The Name property of the tree itself is set to the empty string.`

![image](https://user-images.githubusercontent.com/20224954/115867439-050da600-a433-11eb-9d13-a385aeb1807a.png)

How does one set a name attribute for the tree? My wix code is here: https://github.com/adoptium/installer/blob/master/wix/Main.wxs.template

**Comment by robmen** [comment, marked as answer]
IIRC, you can't. The Windows Installer doesn't expose any way to control the names of many of the controls in the MSI UI. It's been a source of frustration for a couple of decades. Windows needs to update the Windows Installer engine to allow better accessibility of MSI UI controls.
